sites/wonderland/docs/processes/github/git-environment.md

```
@@ -107,14 +107,14 @@ cat ~/.ssh/personal/id_ed25519.pub
 cat ~/.ssh/defi/id_ed25519.pub
 ```
 
-Add each public key to the corresponding GitHub account under Settings → SSH Keys
+Add each public key to the corresponding GitHub account under Settings → SSH Keys.
 
 ## GPG Key Setup
 
 ### Generate GPG key:
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)